test/modellib/MatMulModel.cpp

```diff
@@ -115,28 +115,26 @@ bool MatMul2DLibBuilder::verifyOutputs() {
 // Matmul with broadcast in A or B but not both.
 
 MatMulSingleBroadcastLibBuilder::MatMulSingleBroadcastLibBuilder(
-    const std::string &modelName, bool broadcastingB,
+    const std::string &modelName, bool broadcastingB, bool sameStaticBroadcast,
     std::vector<int64_t> broadcastDims, const int I, const int J, const int K)
     : ModelLibBuilder(modelName), broadcastingB(broadcastingB),
-      broadcastDims(broadcastDims), I(I), J(J), K(K) {}
+      broadcastDims(broadcastDims), sameStaticBroadcast(sameStaticBroadcast),
```

Comment:
fyi, this caused a reorder-ctor compiler warning because sameStaticBroadcast is declared after sameStaticBroadcast in the class definition

I added a fix in PR #1644

Reply:
Yes, I saw that recently. I reordered them in one of my later PR... if conflicts arises during the 2nd merge... we will just deal with them. Thanks for the fix.